New Reagent, Universalizes Survival Medipen

Ignari-contrib opened the pull request:
Adds an alternate regenerative jelly for the survival medipens used in mining. Replaces the tricord and omnizine in medipens with 15u of Neo Jelly. Makes medipens finally useful for slime miners. When ODed on Neo Jelly, causes heavy brute/burn and minor oxyloss and causes no toxin damage.

add: new reagent: Neo Jelly, is almost exactly like Regenerative Jelly, except has an OD threshold of 30u to prevent abuse.
balance: With the removal of tricord and omnizine and replacement with Neo Jelly, this allows for slime miners to have a reason to value medipens just the same as other non slime miners would.
code: removed tricord and omnizine from survival medipens and replaces it with 15u of Neo Jelly.
why: Several slime players who mine have admitted that medipens are functionally useless to them, and instead would kill them if used. This fixes it so slimes heal just like the other miners.

Commit:
New Reagent, Universalizes Survival Medipen

Adds an alternate regenerative jelly for the survival medipens used in mining. Replaces the tricord and omnizine in medipens with 15u of Neo Jelly. Makes medipens finally useful for slime miners. When ODed on Neo Jelly, causes heavy brute/burn and minor oxyloss and causes no toxin damage.

deathride58 commented on code/modules/reagents/chemistry/reagents/medicine_reagents.dm:
```diff
@@ -1056,6 +1056,32 @@
+    ..()
+
+/datum/reagent/medicine/neo_jelly/overdose_process(mob/living/M)
+    M.adjustToxLoss(0REM, 0)
```
Add a `*` before the REM

Ignari-contrib replied:
Done!

Commit:
code alter

adds * to the REM.

Iamgoofball commented on code/modules/reagents/chemistry/reagents/medicine_reagents.dm:
```diff
@@ -1056,6 +1056,32 @@
+    ..()
+
+/datum/reagent/medicine/neo_jelly/overdose_process(mob/living/M)
+    M.adjustToxLoss(0*REM, 0)
```
wew, why have this here at all

Ignari-contrib replied:
Was unsure what to remove and just zero out.

Commit:
additional code fix

removes the M.adjustToxLoss line, and corrects code text to 52 units injected, from 57.

Ignari-contrib commented on code/modules/reagents/chemistry/reagents/medicine_reagents.dm:
```diff
@@ -1056,6 +1056,31 @@
+    reagent_state = LIQUID
+    metabolization_rate = 1 * REAGENTS_METABOLISM
+    color = "#91D865"
+    overdose_threshold = 30
```
`overdose_threshold = 23`

Ignari-contrib reviewed (commented):
lowers the OD threshold to prevent dual penning being a bit OP.
